tests/test_deploy.py

```diff
@@ -166,6 +166,7 @@ def test_deploy_dmi_transformation(
     rmdir(Path(test_name))
 
 
+@pytest.mark.skip
```

Comment:
This test will be replaced by node and edge in https://github.com/cognitedata/transformations-cli/pull/83